zerver/migrations/0423_alter_realmuserdefault_realm_name_in_notifications_and_more.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 4.1.5 on 2023-01-11 20:21
+
+from django.db import migrations, models
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+# We include a copy of this structure as it was at the time this
+# migration was merged, since future should not impact the migration.
+REALM_NAME_IN_NOTIFICATIONS_AUTOMATIC = 1
+REALM_NAME_IN_NOTIFICATIONS_YES = 2
+REALM_NAME_IN_NOTIFICATIONS_NO = 3
+REALM_NAME_IN_NOTIFICATIONS_CHOICES = [
```

Comment:
This list is not needed in this migration. Only variables for all choices are enough.